Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -124,8 +124,8 @@ class ActivityStreamTest: BaseTestCase {
         navigator.performAction(Action.CloseURLBarOpen)
         navigator.performAction(Action.OpenNewTabFromTabTray)
 
-        waitForExistence(app.collectionViews.cells.staticTexts[newTopSite["bookmarkLabel"]!])
-        XCTAssertTrue(app.collectionViews.cells.staticTexts[newTopSite["bookmarkLabel"]!].exists)
+        waitForExistence(app.collectionViews.cells.staticTexts[newTopSite["topSiteLabel"]!], timeout: TIMEOUT)
+        XCTAssertTrue(app.collectionViews.cells.staticTexts[newTopSite["topSiteLabel"]!].exists)
         checkNumberOfExpectedTopSites(numberOfExpectedTopSites: 6)
 
         navigator.performAction(Action.CloseURLBarOpen)
```

Comment:
After this we need to long press on the top site so that we pin it to the home panel screen... currently we are not doing that and we should so that we check that it is not removed after clearing all data. Not sure why that step is not here 🤔  but it should...
So we are missing:
- long tap on the mozilla top site
- select pin from the menu

Reply:
It's odd that the website isn't pinned on the first attempt using the settings menu. Is it a bug?

Reply:
In the first attempt it is added as a short cut but not actually pinned. When the shortcut is pinned it shows this icon:
![Untitled](https://user-images.githubusercontent.com/1897507/202419880-242d8c59-7d9f-4081-8272-ffdbb0e62015.jpg)

Reply:
Was Firefox used to pin 📌 the website by default when the website was pinned via the settings menu?

Reply:
there is not a pin from settings menu, from that menu you can add to shortcut, then you have to pin by long tapping on the shortcut. In the past there was a different menu in the pageOptionsMenu but this menu was removed.